_source/logzio_collections/_metrics-sources/container-groups-azure-metrics.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Ship Azure Container Instance metrics
+logo:
+  logofile: azure-containerinstances_large.svg
+  orientation: vertical
+data-source: Azure Container Instances
+contributors:
+  - yotamloe
+  - shalper
+shipping-tags:
+  - azure
+
+---
+
+To monitor your Azure service metrics,
+we recommend configuring your services
+to send their metrics to Azure Monitor.
+When you set up Metricbeat using the configuration on this page,
+Metricbeat will collect metrics from Azure PostgreSQLs
```

Comment:
you have:
To monitor your Azure service metrics, we recommend configuring your services to send their metrics to Azure Monitor. When you set up Metricbeat using the configuration on this page, Metricbeat will collect metrics from **Azure PostgreSQLs** and forward them to Logz.io.

It should be: **Azure Container Groups*